modules/programs/antidote.nix

```diff
@@ -46,6 +46,14 @@ in {
       zstyle ':antidote:bundle' file $bundlefile
       staticfile=$HOME/${relToDotDir ".zsh_plugins.zsh"}
       zstyle ':antidote:static' file $staticfile
+
+      if [[ $(uname) == "Darwin" ]] ; then 
+        if [[ $( stat -f "%m" -t "%Y" $bundlefile) -gt $(stat -f "%m" -t "%Y" $staticfile) ]] ; then
+          antidote bundle <$bundlefile >$staticfile
+        fi
+      elif [[ $(stat -c "%Y" $bundlefile) -gt $(stat -c "%Y" $staticfile) ]] ; then
+        antidote bundle <$bundlefile >$staticfile
+      fi
```

Comment:
```suggestion
      if [[ $staticfile -ot $bundlefile ]]; then
        antidote bundle < "$bundlefile" > "$staticfile"
      fi
```

Reply:
since `if [[  ! $staticfile -nt $bundlefile ]]; then` not work in the origin `antidote load` 

`-ot ` is alos not work for nix symbol link

Reply:
Wait, the bundlefile is handled by Nix? Why don't we also generate the staticfile with Nix then? `pkgs.runCommand "zsh_plugins.zsh" {} '' antidote bundle < ${bundleFile} > "$out" ''`